src/managedwriter/stream_connection.ts

```diff
@@ -87,7 +87,16 @@ export class StreamConnection extends EventEmitter {
     this._connection.on('error', this.handleError);
     this._connection.on('close', () => {
       this.trace('connection closed');
-      this.close();
+      this.reconnect();
```

Comment:
Let's try not to reconnect for this particular error here: Aborted with an error message of "Closing the stream because it has been inactive for"

Reply:
now we only reconnect when there are pending writes, so in the case of idle connection, the reconnection is not gonna happen right away, only on the next write.

Reply:
Cool